torch/nn/quantized/modules/functional_modules.py

```diff
@@ -137,7 +137,7 @@ def add(self, x, y):
     r"""Operation equivalent to ``torch.ops.quantized.add(Tensor, float)``"""
     def add_scalar(self, x, y):
```

Comment:
I wonder whether we should drop it and just make a dispatch on "is scalar" inside the op. The difference is that the arguments would be different (passing or not qscale).